cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -682,55 +761,78 @@ public void checkCustomCaSecret(CertificateAuthority ca, Secret certSecret, Secr
             }
         }
 
-        Future<ReconciliationState> kafkaUpgrade() {
+        private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSs) {
+
+            // Get the current version of the cluster
+            KafkaVersion currentVersion = versions.version(Annotations.annotations(kafkaSs).get(ANNO_STRIMZI_IO_KAFKA_VERSION));
+
+            log.debug("Does SS {} Kafka version need to be changed?", kafkaSs.getMetadata().getName());
```

Comment:
Ehh, let's try to avoid using `SS` at least in logs ... `STS` is the official shortcut for StatefulSet if that is what is meant here :-). (and I know, the SS shortcut always keeps poping in my head as well :-o)

Reply:
A very good point! I will do a find replace.

Reply:
We're probably got a ton of logging which uses SS, so we should open an issue for fix those.

Reply:
Ok I will fix the `SS` entries in the `KafkaAssemblyOperator` and then open a PR for the other classes.

Reply:
Thanks. And for the record ... I don't think you need to rename the code occuranes as `kafkaSs` etc. ... the log messages are the important part I guess.

Reply:
Only just saw this message. I went a bit overboard on the find replace and swapped all the variable names too :-S